doc/fornax_setup.md

```diff
@@ -0,0 +1,245 @@
+# Fornax Core and Nodeagent Setting Up Detail
+
+## Overview
+This doc is the detail steps which setting up, configuration and install component for Fornax Serverless Machine.
+
+## Machine Prepare
+If you have a brand new machine, your should install following component to start the machine. (Following is detail steps for AWS EC2 virtual machine).
+
+```script
+sudo apt-get update
+sudo apt install build-essential
+sudo snap install curl  # version 7.81.0
+sudo apt-get install vim
+```
+
+## 1. Install and Setup Fornax Core
+### 1.1 Install Etcd 
+Install etcd by using apt-get
+```script
+sudo apt-get update
+sudo apt-get -y install etcd
+```
+
+Verify etcd
+```script
+sudo systemctl status etcd.service
+sudo systemctl enable etcd
+sudo systemctl restart etcd
+```
+
+Stop etcd.service. (note: we use the local etcd by default fornax serverless, so we need stop etcd.service)
+```script
+sudo systemctl stop etcd.service
+```
+
+### 1.2 Install Fornax Core From Source Code
+#### 1.2.1 Install Golang
+```script
+GOLANG_VERSION=${GOLANG_VERSION:-"1.18"}
+sudo apt -y update
+sudo apt -y install make
+sudo apt -y install gcc
+sudo apt -y install jq
+wget https://dl.google.com/go/go${GOLANG_VERSION}.linux-amd64.tar.gz -P /tmp
+sudo tar -C /usr/local -xzf /tmp/go${GOLANG_VERSION}.linux-amd64.tar.gz
+```
+veify go version
+```script
+export PATH=$PATH:/usr/local/go/bin
+go version
+```
+the result should be: go version go1.18 linux/amd64
+
+make following directory (under yourself home directory)
+```script
+sudo mkdir -p go/bin
+sudo mkdir -p go/src
+```
+
+Add following setting to machine ~/.bashrc (notes: ubuntu is yourself user login directory, remember to replace your's)
+```script
+sudo vi ~/.bashrc
+```
+
+```sh
+export GOPATH="/home/ubuntu/go"
+export GOROOT="/usr/local/go"
+export GOBIN="/home/ubuntu/go/bin"
+export PATH="$PATH:/usr/local/go/bin:/home/ubuntu/go/bin"
+```
+Take effetive
+```script
```

Comment:
I do not think these details are necessary, any go developer should know how to do this. it distract reader

Reply:
Yes, your are right, But some entry level user, lower intermedia level user, and site admin need more detail instruction. I thought that we keep your get_start.md as Professional user use. the user can reference fornax_setup.md as detail if they need.